app/views/users/show.html.erb

```diff
@@ -44,9 +58,7 @@
             </span>
           <% end %>
         </h1>
-        <p class="profile-description" itemprop="description">
```

Comment:
Same here. The content should still be there, we just need to get rid of the `itemprop="description"` part.